portfolio/src/main/webapp/script.js

```diff
@@ -86,9 +86,9 @@ class Portfolio {
         if (res.status === 404){
             alert("Empty datastore.");
         } else if (res.status === 500) {
-            alert('Error: invalid count requested. Fewer than ' + this.numComments + ' comments exist.');
+            alert('Error: invalid count requested.  Fewer than ${this.numComments} comments exist.');
```

Comment:
The delimiters for template strings are backticks (`` ` ``), not single quotes (`` ' ``).